kubeflow/metadata.csv

```diff
@@ -0,0 +1,49 @@
+metric_name,metric_type,interval,unit_name,per_unit_name,description,orientation,integration,short_name,curated_metric
+kubeflow.katib.controller.reconcile.count,count,,,,Number of reconcile loops executed by the Katib controller,0,kubeflow,,
+kubeflow.katib.controller.reconcile.duration.seconds.bucket,count,,second,,Duration of reconcile loops executed by the Katib controller(bucket),0,kubeflow,,
+kubeflow.katib.controller.reconcile.duration.seconds.count,count,,second,,Duration of reconcile loops executed by the Katib controller(count),0,kubeflow,,
```

Comment:
Note that for histograms the `.bucket` and `.count` track **observations** not **seconds**. It's confusing 😞 

Let's remove the unit for the `.bucket` and `.count` metrics everywhere.